index.js

```diff
@@ -553,13 +572,44 @@
 				if (this.role === 'server' && obj['validateServerCall']) {
 					return obj['validateServerCall'].call(obj, remoteCall.name, callContext);
 				}
-
 				return true;
 			} else {
 				throw 'Sync Error';
 			}
 		}
 
+		/**
+		 * Apply function specific custom serverSide validation functions
+		 *
+		 * @param {boolean} validationResult - Result of previous validation step (applyChangesAndValidateCall)
```

Comment:
if this is a boolean can we make naming more obvious? e.g. `isValid` or `isValidResult`. that way things like below `if (!validationResult) {` read more easily / naturally like `if (!isValid) {`